src/ape_pm/compiler.py

```diff
@@ -21,25 +30,35 @@ def get_versions(self, all_paths: List[Path]) -> Set[str]:
     def compile(
         self, filepaths: List[Path], base_path: Optional[Path] = None
     ) -> List[ContractType]:
+        filepaths.sort()  # Sort to assist in reproducing consistent results.
         contract_types: List[ContractType] = []
         for path in filepaths:
-            data = json.loads(path.read_text())
+            try:
+                data = json.loads(path.read_text())
+            except JSONDecodeError as err:
+                logger.error(f"({CompilerError}) {err}")
```

Comment:
what does putting `CompilerError` in the f-string do?

Reply:
Meant to do `CompilerError.__name__` but this should actually raise an error so going to **not** do this.. Thanks.